### ISSUE TITLE
[XamarinProfiler-MAC]: Accessibility: Color Contrast: MAS1.4.3: "Reading data… " text present at top middle of profiler windows of both Allocations/Time Profiler is not having color contrast ratio of 4.5:1.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#417]
### Steps to Reproduce

Scenario 1:
1. Follow the pre-requisite to launch the xamarin profiler application.
2. Select the Available instruments to "Allocations" and invoke next button. Invoke Start Profiling button.
3.This will invoke "Allocations" Profiling page.
4. Verify the color contrast ratio of "Reading data ..." text at top middle of window.

Scenario 2:
1. Follow the pre-requisite to launch the xamarin profiler application.
2. Select the Available instruments to "Time Profiler" and invoke next button. Invoke Start Profiling button.
3.This will invoke "Time Profiler" Profiling page.
4. Verify the color contrast ratio of "Reading data ..." text at top middle of window.

### Pre-Requisite:
1. Install Xamarin Profiler.
2. Launch Visual Studio.
3. Create a project and build the solution.
4. Navigate to Run Menu Tab and Invoke it. Navigate to "Start Profiling" option and invoke it.
5. This will invokes "Xamarin Profiler" application.

### Expected Behavior
Color contrast ratio should be minimum 4.5:1
### Actual Behavior
Color contrast ratio is less than 4.5:1 i.e, 2.4:1
See the screenshots attached
### User Impact
Low vision user will feel difficult in seeing the text
### Screenshots
![7 reading data - allocations](https://user-images.githubusercontent.com/22956130/41591774-4ead800c-73d8-11e8-8534-4031f821453f.png)
![7 reading data time profiler](https://user-images.githubusercontent.com/22956130/41591785-58441702-73d8-11e8-98c6-d7189a19bc8c.png)

### Environment
macOS High Sierra OSX version: 10.13.4
Visual Studio for Mac Enterprise version: 7.5.2(build 40)
Xamarin Profiler version: 1.6.3-400

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS1.4.3
Xamarin-Profiler-MAC
Wipro
